app/src/components/blocks/_lloydGeorge/lloydGeorgeSelectSearchResults/LloydGeorgeSelectSearchResults.tsx

```diff
@@ -51,6 +51,23 @@ const AvailableFilesTable = ({
             setSelectedDocuments(selectedDocuments.filter((id) => id !== toggledDocumentId));
         }
     };
+
+    const handleToggleButtonAriaDescription = () => {
+        if (selectedDocuments.length === searchResults.length) {
+            return 'Toggle selection button, Click to deselect all files';
+        } else {
+            return 'Toggle selection button, Click to select all files';
+        }
+    };
+
+    const handleToggleButtonStatusChange = () => {
+        if (selectedDocuments.length === searchResults.length) {
+            return 'All files are selected';
+        } else if (selectedDocuments.length === 0) {
+            return 'All files are deselected';
+        }
+    };
+
```

Comment:
I would suggest renaming these two functions to something like "getAriaDescription", "getAnnouncementMessage". 
Usually in react we call a function `handleXXX` if it handles an event and triggers an action that affects other things, e.g. make an API call or update a state. Here these two function only provide a different string depending on situation, so probably better not to call them `handleXXX`

Reply:
Done